### PR TITLE
#401 + #445: Fixed size selection bug + Deleting non-selected address

### PR DESCRIPTION
**Pull request by ahmed-shehata**
#401: Fixed size selection bug

### CODE REVIEW

**Comment by ahmed-shehata** [comment]
#445 Fixed Deleting address resets cart

**Comment by ahmed-shehata** [comment]
👍

File: AtlasUI/AtlasUI/SizeListSelectionViewController.swift [hunk 1]
```diff
@@ -92,7 +92,7 @@ extension SizeListSelectionViewController: UITableViewDelegate {
         tableView.deselectRowAtIndexPath(indexPath, animated: true)
 
         guard Atlas.isUserLoggedIn() else {
-            let selectedArticleUnit = SelectedArticleUnit(article: article, selectedUnitIndex: 0)
+            let selectedArticleUnit = SelectedArticleUnit(article: article, selectedUnitIndex: indexPath.row)
```
Comment (hani-ibrahim) [inline AtlasUI/AtlasUI/SizeListSelectionViewController.swift]: Nice catch 👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

**Pull request merged**